dll/win32/shell32/wine/pidl.c

```diff
@@ -1722,6 +1722,80 @@ LPITEMIDLIST _ILCreateGuidFromStrA(LPCSTR szGUID)
 }
 #endif
 
+// Create GUID from shortcut(desktop, Personal...)
+// returns TRUE if succefull and FALSE otherwise
+BOOL _ILCreateGuidFromShortcutW(LPCWSTR szShortcut, GUID *fidOut)
+{
+	if (!StrCmpIW(szShortcut, L"ADMINISTRATIVE TOOLS"))
+	{
+		*fidOut = CLSID_AdminFolderShortcut;
+		return TRUE;
+	}
+	else if (!StrCmpIW(szShortcut, L"CONTROLPANELFOLDER"))
```

Comment:
Also I noticed you used tabs for indentation. Please use 4-space (this is mentioned in our code guidelines).